packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
   propertyIndex: number;
   onRemove: () => void;
 }) => {
-  const { watch } = useFormContext<EntityTypeEditorForm>();
+  const { control, watch } = useFormContext<EntityTypeEditorForm>();
+  const { update } = useFieldArray({
+    control,
+    name: "properties",
+  });
+
   const propertyTypes = usePropertyTypes();
-  const propertyId = mustBeVersionedUri(
-    watch(`properties.${propertyIndex}.$id`),
-  );
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null);
+  const [multipleValuesMenuOpen, setMultipleValuesMenuOpen] = useState(false);
+
+  const draftProperty = watch(`properties.${propertyIndex}`);
```

Comment:
Generally `watch` should be avoided or used at as low a level as possible and on as specific a property path as possible. `react-hook-form`'s philosophy is about avoiding re-renders when inputs change value by keeping components uncontrolled, and this watch will break that and make the entire row re-render when any value within the property changes.  

It also turns out we should `useWatch`, not `watch`, for performance.  https://react-hook-form.com/api/usewatch

Reply:
Replaced `watch` with `useWatch` as part of 669f2a5700d2e0a6385302b661eb077a24186144.